### DISCUSSION TITLE
Feature Request: CYGWIN cygsshd service protection

### DISCUSSION THREAD

**Discussion by warichnict** [DigitalRuby/IPBan#160] [Ideas]
Hi,

first of all thanks a lot for your grate work with IPBan, i am running a windows server with open RDP service to the internet, and even then i am sure no one would guess any password on that system i feeling much relieved that all that constantly probing IPs are now blocked.

But i also run the ssh server of the cygwin project (see https://gist.github.com/roxlu/5038729 for install instructions) which is grate cause i can to a degree log into and work with my windows server like i am used to on gnu/linux systems, there i use fail2ban to block ips probing it, but i still need a way to protect my cygsshd on windows, so it would be really grate if you could implement that.

Thanks a lot in advance, and keep up the good work!

**Comment by jjxtra** [comment]
Need log file paths with example lines of failed and success logins. FYI ipban works great on Linux ;)
